packages/ember-htmlbars/tests/helpers/unbound_test.js

```diff
@@ -1,595 +0,0 @@
-import EmberView from 'ember-views/views/view';
-import EmberComponent from 'ember-views/components/component';
-import EmberObject from 'ember-runtime/system/object';
-
-import { A as emberA } from 'ember-runtime/system/native_array';
-import Ember from 'ember-metal/core';
-import { get } from 'ember-metal/property_get';
-import { set } from 'ember-metal/property_set';
-import run from 'ember-metal/run_loop';
-import compile from 'ember-template-compiler/system/compile';
-import Helper, { helper } from 'ember-htmlbars/helper';
-
-import { runAppend, runDestroy } from 'ember-runtime/tests/utils';
-
-import buildOwner from 'container/tests/test-helpers/build-owner';
-import { OWNER } from 'container/owner';
-
-var view, lookup, owner;
-var originalLookup = Ember.lookup;
-
-QUnit.module('ember-htmlbars: {{#unbound}} helper', {
-  setup() {
-    Ember.lookup = lookup = { Ember: Ember };
-
-    view = EmberView.create({
-      [OWNER]: owner,
-      template: compile('{{unbound foo}} {{unbound bar}}'),
-      context: EmberObject.create({
-        foo: 'BORK',
-        barBinding: 'foo'
-      })
-    });
-
-    runAppend(view);
-  },
-
-  teardown() {
-    runDestroy(view);
-    runDestroy(owner);
-    owner = view = null;
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('it should render the current value of a property on the context', function() {
-  equal(view.$().text(), 'BORK BORK', 'should render the current value of a property');
-});
-
-QUnit.test('it should not re-render if the property changes', function() {
-  run(function() {
-    view.set('context.foo', 'OOF');
-  });
-  equal(view.$().text(), 'BORK BORK', 'should not re-render if the property changes');
-});
-
-QUnit.test('it should not re-render if the parent view rerenders', function() {
-  run(function() {
-    view.set('context.foo', 'OOF');
-    view.rerender();
-  });
-  equal(view.$().text(), 'OOF OOF', 'should re-render if the parent view rerenders');
-});
-
-QUnit.test('it should assert unbound cannot be called with multiple arguments', function() {
-  expectAssertion(function() {
-    runAppend(EmberView.create({
-      template: compile('{{unbound foo bar}}'),
-      context: EmberObject.create({
-        foo: 'BORK',
-        bar: 'foo'
-      })
-    }));
-  }, /unbound helper cannot be called with multiple params or hash params/);
-});
-
-QUnit.test('should property escape unsafe hrefs', function() {
```

Comment:
Is this replaced by `should render on attributes`? If so, why is the `unsafe:` part changed?